### PR TITLE
PEAR-1840: Responsive cohort builder

### PR DESCRIPTION
**Pull request by paribartandhakal**
## Description
Some styling changes in cohort builder page to make UI more reponsive.

## Checklist
- [N/A] Added proper unit tests
- [N/A] Left proper TODO messages for any remaining tasks
- [N/A] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

File: packages/portal-proto/src/components/BtnWithHoverCallout.tsx [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+  flex items-center w-full text-left
+  hover:bg-primary-lightest
+  hover:after:content-['']
+  hover:after:absolute
+  hover:after:top-[-8px]
+  hover:after:left-1/2
+  hover:after:transform
+  hover:after:translate-x-[-50%]
+  hover:after:border-l-[8px]
+  hover:after:border-l-solid
```
Comment (amlehman) [inline packages/portal-proto/src/components/BtnWithHoverCallout.tsx]: These border solid styles are overwritten on the next line by the border transparent (I see the original component had this mistake too which I think is my bad 🙀 ).

**Review by amlehman — APPROVED** [review]

File: packages/portal-proto/src/components/BtnWithHoverCallout.tsx (hunk 1 above)
Comment (paribartandhakal) [reply packages/portal-proto/src/components/BtnWithHoverCallout.tsx]: I just updated. Yes I overlooked at it too. TBH I am still confused with that css 😛 . I verified that it still works as expected.

**Comment by wteouchicago** [comment]
I mostly looked at the tooltip part for search and it looks good. 👍

File: packages/portal-proto/src/components/BtnWithHoverCallout.tsx (hunk 1 above)
Comment (amlehman) [reply packages/portal-proto/src/components/BtnWithHoverCallout.tsx]: It's a trick to create a triangle using CSS: https://css-tricks.com/snippets/css/css-triangle/

Comment (paribartandhakal) [reply packages/portal-proto/src/components/BtnWithHoverCallout.tsx]: Oh wow thank you. I will look at it.

**Comment by cemile-ctds** [comment]
Looks good on my side.

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**